### ISSUE TITLE
fix left nav menu style on RTD

### ISSUE THREAD

**Issue by mradamcox** [archesproject/arches-docs#258]
#### describe the issue
On Read the Docs (but not in local builds) the margins are off for headers in the left-hand navigation menu.

This:

![2021-08-02_16-00-menu-issue](https://user-images.githubusercontent.com/10427268/127923370-49f874aa-b998-4fb7-b56c-a72193ade3ec.png)

Should look like this:

![2021-08-02_16-00-menu-issue-good](https://user-images.githubusercontent.com/10427268/127923471-baeaa197-8f36-433e-a6da-bc078baf42ac.png)

The underlying issue seems to be that the `<p>` elements in RTD aren't getting the `caption` class added to them, but they do in local builds. Most likely some environmental settings are to blame.

##### which release does this issue concern?
5.1 (stable) and latest. **5.0 and lower are fine.**

**Comment by mradamcox** [comment]
I found this ticket on the sphinx_rtd_theme repo https://github.com/readthedocs/sphinx_rtd_theme/issues/1112 which discusses the underlying issue. To fix it I added a .readthedocs.yaml file to the latest branches (`master` and `5.1`) of this repo, which tells RTD to use the `requirements.txt` file on deployment, and then pinned a later release of the RTD theme in there. These updates have fixed the issue.
